components/PerangkatDesa/Detail/index.vue

```diff
@@ -0,0 +1,378 @@
+          />
+        </div>
+        <div v-else>
+          <BaseButton
+            label="Kembali"
+            variant="secondary"
+            @click="onClose"
+          />
+          <BaseButton
+            label="Kirim Sekarang"
+            variant="primary"
+            @click="sentRejectPerangkatDesa"
+          />
+        </div>
+      </div>
+    </BaseCustomDialog>
+  </div>
+</template>
+
+<script>
+import IconArrowLeft from '~/assets/icons/arrow-left.svg?inline'
+import IconPdf from '~/assets/icons/IconPdf.svg?inline'
+import { formatDateTime } from '~/utils'
+import { userStatus } from '@/constants/dataUser'
+
+export default {
+  name: 'ComponentPerangkatDesaDetail',
+  components: { IconArrowLeft, IconPdf },
+  props: {
+    item: {
+      type: Object,
+      default: () => ({})
+    }
+  },
+  data () {
+    return {
+      mItem: {},
+      userStatus,
+      formatDateTime,
+      dataPerangkatDesa: null,
+      isDialog: false,
+      contentPerangkatDesa: {
+        header: '',
+        title: '',
+        name: '',
+        data: '',
+        notes: '',
+        showNotes: false,
+        lengthNotes: 5000,
+        error: null
+      }
+    }
+  },
+  watch: {
+    item: {
+      handler (val) {
+        this.mItem = {
+          ...val
+        }
+      },
+      immediate: true,
+      deep: true
+    },
+    'contentPerangkatDesa.notes' () {
+      const maxLength = 5000
+      this.contentPerangkatDesa.lengthNotes = maxLength - this.contentPerangkatDesa.notes.length
+      if (this.contentPerangkatDesa.notes.length > 1) {
+        this.contentPerangkatDesa.error = null
+      }
+    }
+  },
+  methods: {
+    refreshData () {
+      this.$emit('updated')
+    },
+    goBack () {
+      this.$store.commit('setSubMenu', 'PerangkatDesa')
+      this.$store.commit('setTrackPage', true)
+      this.$router.push('/data-user')
+    },
+    deleteDataPartner (item) {
+      this.$store.dispatch('dialog/showDialog', {
+        header: 'Hapus Perangkat Desa',
+        title: 'Apakah Anda yakin akan menghapus data perangkat desa ini?',
+        message: `${item.name} - ${item.village.name}`,
+        btnRightVariant: 'danger',
+        btnLeftVariant: 'secondary',
+        actionBtnRight: () => this.actiondeleteDataPartner(item)
+      })
+    },
+    async actiondeleteDataPartner (item) {
+      try {
+        const response = await this.$axios.delete(`/users/${item.id}`)
+        if (response) {
+          this.$store.dispatch('dialog/showDialog', {
+            header: 'Hapus Data Perangkat Desa Berhasil',
+            title: 'perangkat desa ini berhasil dihapus.',
+            message: `${item.name} - ${item.village.name}`,
+            dialogType: 'information',
+            iconMessage: 'trash',
+            iconColor: 'text-red-500',
+            btnLeftLabel: 'Saya Mengerti',
+            btnLeftVariant: 'primary',
+            actionBtnLeft: () => this.$router.push('/data-user')
+          })
+        }
```

Comment:
minor/ask:

Hello, @doohanas is this conditional statement necessary?
```js
if (response) {
  ...
}
```
if the fetching data is successful, the response variable will always have a value right? 🤔

Reply:
Hello @Ibwedagama, yes, after fetching data is successful., the response will have value. So, it's okay when we remove the conditional, wdyt?

Reply:
In my opinion, it's safe to remove the conditional because of how the try-catch block works, what's your opinion about this @yoslie ?

Reply:
Please check the changes [here](https://github.com/jabardigitalservice/dedi-cms/pull/87/commits/5b4ff9f4038e4d533bf0e2b1212b6691348e5cc9), thank you ⭐